keyboards/sofle/keymaps/bastien-baloup_fr/keymap.c

```diff
@@ -0,0 +1,218 @@
+ /* Copyright 2022 Bastien Baloup
+  *
+  * This program is free software: you can redistribute it and/or modify
+  * it under the terms of the GNU General Public License as published by
+  * the Free Software Foundation, either version 2 of the License, or
+  * (at your option) any later version.
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+    PLACEHOLDER = SAFE_RANGE,  // can always be here (4 bytes)
+    // CL_ keycodes are here for charaters that need capslock on to be inputed.
+    CL_AGRV,                   // À
```

Comment:
You don't need the "placeholder" there. And in fact, it wastes space. Not much, but if you're counting bytes...
```suggestion
    // CL_ keycodes are here for charaters that need capslock on to be inputed.
    CL_AGRV = SAFE_RANGE,      // À
```